packages/dispatchr/lib/DispatcherContext.js

```diff
@@ -37,7 +37,11 @@ DispatcherContext.prototype.getStore = function getStore(name) {
     if (!this.storeInstances[storeName]) {
         var Store = this.dispatcher.stores[storeName];
         if (!Store) {
-            throw new Error('Store ' + storeName + ' was not registered.');
+            if ('function' === typeof name) {
```

Comment:
yeah I mean 

ideally we should make stores registered, if there's any case that user getStore without register it, then there must be something they need to fix.

previously we let it throw, now if we handle this, people would never aware of the issue.